### PR TITLE
5.2.2 version history

### PR DESCRIPTION
**Pull request by hflynn**
Will be staged at https://www.openmicroscopy.org/site/support/omero5.2-staging/users/history.html but not until the autogen branches have been updated so initial review should be of the text and I'll check the formatting later (although it should be fine).

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+-  OMERO.web fixes to improve synchronicity when deleting many images
+-  better handling of dates in search
+-  client support for map annotations in OME.tiff
+-  fixes for displaying bulk annotations script in OMERO.insight
+-  OMERO.web clean-up to remove defunct volume viewer
+
+Developer updates include:
+
+-  OMERO.tables documentation extended
+-  updating 'What's New for developers' to clarify that ``pogos`` has been
```
Comment (mtbc) [inline history.txt]: Should this be "pojos"? (I really don't know.)

**Comment by jburel** [comment]
This PR should be mentioned https://github.com/openmicroscopy/openmicroscopy/pull/4398
i.e. improve delete/change group performance

**Comment by pwalczysko** [comment]
Missing mention about the support for masks in Web see https://github.com/openmicroscopy/openmicroscopy/pull/4432

**Comment by will-moore** [comment]
I think for web, you can remove these:
 - support for units in the OMERO.web image viewer scalebar
 -  OMERO.web fixes to improve synchronicity when deleting many images
Also remove "better handling of dates in search" *if* that's just referring to web?

And use these in their place (I don't know if you want to add a Features section)?

Features:
 - Display of ROI masks in the OMERO.webclient image viewer
 - Display of OMERO.tables data for Wells in the webclient right panel
 - An new script "Populate Metadata" to enable generation of OMERO.tables for Wells from the web and Insight clients.
 
Bug fixes:
 - Pixel size metadata and scalebar fixed in web viewer for images with pixel size units other than micrometer.
 - Previous & Next buttons in web image viewer.
 - Fixed webclient's handling of ```$ bin/omero config set omero.client.viewer.interpolate_pixels false```
 - A number of other minor webclient bug-fixes.

Developer updates:
 - Python API examples for creating Polygon and Mask shapes
 - Python API example for "Populate Metadata" to create OMERO.tables for Wells.

Maybe sounds better to use "obsolete volume viewer" instead of "defunct"?

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+Developer updates include:
+
+-  OMERO.tables documentation extended
+-  updating 'What's New for developers' to clarify that ``pojos`` has been
+   renamed as ``omero.gateway.model``
+-  dynamic scripts functionality documented
+
+System administrator updates include:
+
+-  new configurations for fixing OMERO.web download issues
```
Comment (aleksandra-tarkowska) [inline history.txt]: Could we rewrite it as say, "clarify/cleanup OMERO.web doc, including experimental solution to resolve download issue"?

**Comment by aleksandra-tarkowska** [comment]
> Features:

- Disabled orphaned container feature

I am about to add that to the public repository page...

**Comment by hflynn** [comment]
Update pushed, thing I've incorporated everything suggested apart from the metadata script API stuff which i thought @jburel commented somewhere wasn't going to be merged for 5.2.2.

File: history.txt [hunk 3]
```diff
@@ -1,6 +1,45 @@
...
+-  disabled orphaned container feature
+-  OMERO.web clean-up to remove obsolete volume viewer
+
+Developer updates include:
+
+-  Python API examples for creating Polygon and Mask shapes
+-  OMERO.tables documentation extended
+-  updated 'What's New for developers' to clarify that ``pojos`` has been
+   renamed as ``omero.gateway.model``
+-  dynamic scripts functionality documented
```
Comment (aleksandra-tarkowska) [inline history.txt]: sorry forgot about that: dynamically loading omero.client server settings into HTTP session

**Comment by will-moore** [comment]
I was talking about https://github.com/openmicroscopy/openmicroscopy/pull/4455, which is merged. @jburel commented on https://github.com/openmicroscopy/openmicroscopy/pull/4426, but this is actually merged too.

**Comment by hflynn** [comment]
Ah, apologies for that confusion, point added now, along with @aleksandra-tarkowska dynamic loading

**Comment by hflynn** [comment]
(PR closing and re-opening was to kick Travis BTW, this keeps randomly failing the python build for some reason)

**Comment by joshmoore** [comment]
Sounds impressively good from my point of view. I do wonder if we're approaching the point where we need more breakdown (i.e. it's getting long) but probably just something to keep in mind going forward.

**Comment by hflynn** [comment]
More sections you mean? I can split out new functionality and bug fixes as Will suggested if you prefer?

**Comment by joshmoore** [comment]
I can be happy with or without a change. An alternative is to use both frequency of releases and decoupling will tend to reduce the number of things here over time.

**Comment by hflynn** [comment]
I'll default to leaving it this time then, I figured the distinction between new client features and client bug fixes was pretty semantic for our users. Will did actually suggest removing some of the bug fixes and just listing them as 'other bug fixes' but it didn't seem worth selling ourselves short for the sake of brevity.

If there are no further comments, I'll convert to a forum post and news item tomorrow ready for the release.

File: history.txt [hunk 4]
```diff
@@ -1,6 +1,48 @@
...
+   Wells
+-  OMERO.tables documentation extended
+-  updated 'What's New for developers' to clarify that ``pojos`` has been
+   renamed as ``omero.gateway.model``
+-  dynamic scripts functionality documented
+-  dynamic loading of omero.client server settings into HTTP sessions
+
+System administrator updates include:
+
+-  clarification of OMERO.web documentation, including an experimental
```
Comment (aleksandra-tarkowska) [inline history.txt]: clarification of OMERO.web documentation for nginx deployment,

**Comment by hflynn** [comment]
Content (with new commit suggested by Ola) copied to draft forum post and news item, flagged on relevant Trello release board cards.

File: history.txt [hunk 5]
```diff
@@ -1,6 +1,48 @@
...
+   Wells is now usable from both OMERO.web and OMERO.insight (note this is
+   still in development and has some limitations)
+-  measurement tool fixes
+-  fixed pixel size metadata and scalebar in OMERO.web image viewer for images
+   with pixel size units other than micrometer
+-  fixed OMERO.web handling of turning off interpolation of pixels
+-  previous and next buttons fixed in OMERO.web image viewer
+-  delete and change group performance improvements
+-  better handling of dates in search
+-  client support for map annotations in OME.tiff
```
Comment (pwalczysko) [inline history.txt]: OME-TIFF

**Comment by hflynn** [comment]
Fix pushed and updated in forum post & news item too.

**Comment by jburel** [comment]
@hflynn looks good. Thanks. Merging

**Comment by joshmoore** [comment]
Added under mainline for discussion today (and sorry for realizing late!): might be good to add "Upgrade to Bio-Formats 5.1.8"

**Comment by hflynn** [comment]
Bah, I forgot to open a PR to add that yesterday, will do so later. In the meantime, I'm updating the history in the docs and re-releasing them today with Ola's web dependencies on upgrade PR.

**Comment by hflynn** [comment]
Ah, @jburel had already opened https://github.com/openmicroscopy/openmicroscopy/pull/4500

**Pull request merged**